src/cpp/client/channel_cc.cc

```diff
@@ -111,17 +111,17 @@ internal::Call Channel::CreateCall(const internal::RpcMethod& method,
         context->propagation_options_.c_bitmask(), cq->cq(),
         method.channel_tag(), context->raw_deadline(), nullptr);
   } else {
-    const char* host_str = nullptr;
+    const string* host_str = nullptr;
     if (!context->authority().empty()) {
```

Comment:
Use context->authority_ here too? Or should compiler be able to optimize it?

Reply:
@yang-g I made this change in the last commit. PTAL.